spec/imsc-hrm.html

```diff
@@ -247,9 +247,164 @@ <h2>Terms and Definitions</h2>
+        synchronized with the author-specified display times.</p>
+  
+      <p>Playback systems include desktop computers, mobile devices and
+        home theatre devices.</p>
+
+      <p class="note">The HRM is not a new concept: it has been included in all
+      versions and editions of [[IMSC]] and has remained substantially
+      unchanged. It is refactored herein to simplify document maintenance.</p>
+    </section>
+  
+    <section>
+      <h3>Why limit the complexity of <a>IMSC Document Instances</a>?</h3>
+  
+      <p><a>IMSC Document Instances</a> are typically authored by a first party
+      and rendered by a second party. Unless both parties agree on the maximum
+      complexity of a <a>IMSC Document Instance</a>, it is likely that:</p>
+  
+      <ul>
+        <li>an <a>IMSC Document Instance</a> authored by a first party will
+        exceed the capabilities of the <a>presentation processor</a> of a second
+        party, resulting in an incomplete presentation where some subtitles or
+        captions might be missing or might be presented too late;</li>
+
+        <li>a first party authors only extremely simple <a>IMSC Document
+        Instance</a> in an attempt to ensure a complete presentation across all
+        renderers, resulting in a lower quality presentation;</li>
+  
+        <li>a second party over-provisions their <a>presentation processor</a>
+        in order to ensure a complete presentation of all <a>IMSC Document
+        Instances</a>, resulting in increased run-time resource usage, code
+        complexity, etc.; or</li>
+  
+        <li>one <a>presentation processor</a> becomes the de-facto reference to
+        determine whether the presentation of a <a>IMSC Document Instance</a>
+        will succeed, at the expense of other renderers and consistency of
+        presentation.</li>
+      </ul>
 
-    <section class="informative">
-      <h2>Overview</h2>
+      <figure id='fig-intro'>
+        <img src="figures/intro-fig-1.svg" alt="The HRM prevents incomplete presentations of IMSC Instance Documents.">
+
+        <figcaption>
+          The HRM allows authors and implementers of <a>presentation
+          processors</a> to agree on the maximum complexity of <a>IMSC Document
+          Instances</a>.
+        </figcaption>
+      </figure>
+
+      <p>As illustrated in <a href="#fig-intro"></a>, by defining a method (the
+      HRM) to compute a proxy for the complexity of an <a>IMSC Document
+      Instance</a> and specifying a complexity limit based on such proxy:</p>
+
+      <ul>
+        <li>authors can ensure that their <a>IMSC Document Instances</a> do not
+        exceed this limit;</li>
+  
+        <li>implementer can design their <a>presentation processor</a> to maximize the
+        likelihood that it will be able to render correctly all <a>IMSC Document
+        Instances</a> that do not exceed this limit; and</li>
```

Comment:
minor nit for consistency with the other bullets:

```suggestion
        <li>implementers can design their <a>presentation processors</a> to maximize the
        likelihood that they will be able to render correctly all <a>IMSC Document
        Instances</a> that do not exceed this limit; and</li>
```